lib/service/config.go

```diff
@@ -29,6 +31,9 @@ type Config struct {
 	MaxReceiveAmount      int64  `envconfig:"MAX_RECEIVE_AMOUNT" default:"0"`
 	MaxSendAmount         int64  `envconfig:"MAX_SEND_AMOUNT" default:"0"`
 	MaxAccountBalance     int64  `envconfig:"MAX_ACCOUNT_BALANCE" default:"0"`
+	LnurlDomain           string `envconfig:"LNURL_DOMAIN" default:"example.com"`
+	LnurlAPIPrefix        string `envconfig:"LNURL_API_PREFIX" default:"ln"`
```

Comment:
I think it's better when `LnurlAPIPrefix` and `LnurlDomain` are combined into a single variable (`LnurlDomain`).

All occurences in the code use `LnurlAPIPrefix` and `LnurlDomain` together (as `LnurlAPIPrefix + "." + LnurlDomain`) except this one:

``` golang
responseBody.Metadata = lnurlDescriptionHash(user.Nickname, controller.svc.Config.LnurlDomain)
```

and even there I think the full domain name should be used (including prefix).

Reply:
cherry-picked!